### PR TITLE
New script idr_copy_plate.py

### PR DESCRIPTION
**Pull request by will-moore**
This is the script I used to copy plate from IDR http://idr.openmicroscopy.org/webclient/?show=plate-422 to create plate on outreach http://outreach.openmicroscopy.org/webclient/?show=plate-55

NB: I renamed channels and tweaked rendering settings afterwards in webclient.

### CODE REVIEW

File: maintenance/scripts/idr_copy_plate.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+import omero
+from omero.gateway import BlitzGateway
+from omero.rtypes import rint, rstring
+from omero.model import PlateI
+
+USERNAME = "trainer-1"
+PASSWORD = "password"
+# The plate we want to copy
+plage_id = 422
```
Comment (pwalczysko) [inline maintenance/scripts/idr_copy_plate.py]: plage ? (not plague ?) :)

**Comment by will-moore** [comment]
@pwalczysko You should be able to run this via CLI now
```$ python idr_copy_plate.py username password idr_plate_id```

**Comment by joshmoore** [comment]
It occurs to me that slightly longer-term it would be interesting to address how all of these scripts could be made OMERO.cli-compatible.

**Comment by pwalczysko** [comment]
Tested on outreach, user-1. Works as expected.

**Comment by jburel** [comment]
With the various sessions coming, I would think that we will have a good starting point in few months time

**Pull request merged**